project/api/models/civi.py

```diff
@@ -190,8 +197,10 @@ def score(self, request_acct_id=None):
             votes_total = votes['total'] if votes['total'] > 1 else 2
 
             #step3 - C
-            if abs(x) / v <= 5:
-                rank = abs(scores_sum) * math.log10(votes_total) * amp + y + f + g / time_ago
+
+            if abs(scores_sum) / votes_total <= 5.0:
+                rank = abs(scores_sum) * math.log10(votes_total) * amp + scores_sum + favorite + gravity / time_ago
```

Comment:
While we're here, lets see if we can give `amp` a slightly more meaningful name. If not, it's alright, but this may be an easy readability improvement.